util/src/main/java/com/farao_community/farao/util/SystematicSensitivityAnalysisService.java

```diff
@@ -101,7 +103,7 @@ private static void buildFlowFromNetwork(Network network, Crac crac, Map<Cnec, D
         states.forEach(state -> crac.getCnecs(state).forEach(cnec -> {
             try {
                 cnecMarginMap.put(cnec, cnec.computeMargin(network));
-                Optional<Double> maxThreshold = cnec.getThreshold().getMaxThreshold();
+                Optional<Double> maxThreshold = cnec.getThreshold().getMaxThreshold(MEGAWATT);
```

Comment:
I think the SystematicSensitivityAnalysisService works with AMPERE everywhere.

I would put AMPERE, and delay discussions on what should be the units of the systematic sensi for EPIC 4, hoping it will not bother us until then

Reply:
ok